docs/book/glossary.md

```diff
@@ -7,33 +7,36 @@ concepts, their canonical name, and how they relate to or differ from other term
+| inconsistency proof               | set of several recovery symbols with their Merkle proofs such that the decoded *sliver* does not match the corresponding hash; this proves an incorrect/inconsistent encoding by the client |
+| member                            | an *SN* that is part of a *committee* at some *epoch*                                                                                                                                       |
+| permanent blob                    | blob which cannot be deleted by its owner and is guaranteed to be available until at least its expiry epoch (assuming it is valid)                                                          |
+| point of availability (PoA)       | point in time when a *CoA* is submitted to Sui and the corresponding *blob* is guaranteed to be available until its expiration                                                              |
 | publisher                         | service interacting with Sui and the *SNs* to store *blobs* on Walrus; offers a simple `HTTP POST` endpoint to *end users*                                                                  |
-| aggregator                        | service that reconstructs *blobs* by interacting with *SNs* and exposes a simple `HTTP GET` endpoint to *end users*                                                                         |
-| cache                             | an *aggregator* with additional caching capabilities                                                                                                                                        |
-| (Walrus) client                   | entity interacting directly with the *SNs*; this can be an *aggregator*/*cache*, a *publisher*, or an *end user*                                                                            |
-| (blob) reconstruction             | decoding of the primary *slivers* to obtain the blob; includes re-encoding the *blob* and checking the Merkle proofs                                                                        |
+| RedStuff                          | our erasure-encoding approach, which uses two different encodings (*primary* and *secondary*) to enable shard recovery; details are available in the [whitepaper](./walrus.pdf)             |
+| shard                             | (disjoint) subset of erasure-encoded data of all *blobs*; at every point in time, a *shard* is assigned to and stored on a single *SN*                                                      |
+| sliver                            | erasure-encoded data of one *shard* corresponding to a single blob for one of the two encodings; this contains several erasure-encoded symbols of that blob but not the *blob metadata*     |
 | (shard/sliver) recovery           | process of an *SN* recovering a *sliver* or full *shard* by obtaining recovery symbols from other *SNs*                                                                                     |
```

Comment:
Suggestion: Change this entry to just `recovery` and slightly change the description:
```suggestion
| recovery           | process of an *SN* recovering one or multiple *blobs*, *slivers*, or full *shards* by obtaining recovery symbols from other *SNs*                                                                                     |
```